app/src/main/java/me/ash/reader/infrastructure/android/MainActivity.kt

```diff
@@ -45,12 +46,10 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        WindowCompat.setDecorFitsSystemWindows(window, false)
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            window.addFlags(FLAG_LAYOUT_IN_SCREEN or FLAG_LAYOUT_NO_LIMITS)
-        }
         Log.i("RLog", "onCreate: ${ProfileInstallerInitializer().create(this)}")
 
+        enableEdgeToEdge()
```

Comment:
This should be called [before `onCreate()`](https://developer.android.com/develop/ui/views/layout/edge-to-edge#enable-edge-to-edge-display)

Reply:
don't forget `CrashReportActivity`!

Reply:
`enableEdgeToEdge()` only require to be called before setContentView.
it's ok after `onCreat()`